Changelog.md

```diff
@@ -25,7 +25,7 @@ This changelog keeps track of work items that have been completed and are ready
 
 ### Fixes
 
-- **General**: TODO ([#TODO](https://github.com/kedacore/http-add-on/issues/TODO))
+- **RBAC**: Narrow permissions given to components ([#612](https://github.com/kedacore/http-add-on/issues/612))
```

Comment:
This might be more customer-friendly so they understand what has changed?
```suggestion
- **RBAC**: Introduce fine-grained permissions per component and reduce required permissions ([#612](https://github.com/kedacore/http-add-on/issues/612))
```